matching.py

```diff
@@ -0,0 +1,248 @@
+import pandas as pd
+from sklearn.preprocessing import MultiLabelBinarizer
+
+#define max mentees/Users/ijac/side-projects/lenny_mentorship/Mentee_for_matching.csv /Users/ijac/side-projects/lenny_mentorship/Mentor_for_matching.csv
+max_mentees_per_mentor = 2
+
+#load data
+mentors = pd.read_csv('Mentor_for_matching.csv')
+mentees = pd.read_csv('Mentee_for_matching.csv')
+
+
+#Clean DF
+#define function to merge columns with same names together
+def same_merge(x): return ','.join(x[x.notnull()].astype(str))
+
+#define new DataFrame that merges columns with same names together
+mentees = mentees.groupby(level=0, axis=1).apply(lambda x: x.apply(same_merge, axis=1))
+
+print("mentor columns:", mentors.columns.values)
+print("mentee columns:", mentees.columns.values)
+
+mentors_flitered = mentors.filter(items=["Email",
+                 "Offset",
+                 'In-Person Meeting Location',
+                 "Avg Year of YOE",
```

Comment:
@ijac13 Everything looks fine but I'm not sure where "Avg Year of YOE" comes from I don't see it in the output file from Coda so you may just be creating a bunch of nulls